### ISSUE TITLE
Homepage

### ISSUE THREAD

**Issue by joelhenriquez23** [silk-worldwide/signalboosters-2017#5]
Link to Production Files: https://drive.google.com/drive/folders/0BwElxc47J15_Vnlfb21rb010ZW8?usp=sharing

**Comment by dark-ang3l** [comment]
These sections: https://www.screencast.com/t/O7xZOQb4EJHM from the homepage, we assume are going to be hardcoded and we can start creating the design for these. 

What about this section: https://www.screencast.com/t/r7ZUVJcvfj8 ?
Will this be a hardcoded section, or should it dynamical change based on admin panel modifications?
There are 4 types of pages in there, blog, plus other 3 tabs for different web pages. How exactly would you like those to be split the web pages?

**Comment by joelhenriquez23** [comment]
Yes, the "Explore Solutions by Brand" section should be hardcoded.

The "Information & Resources" section should be dynamic...we expect these component to have fresh content. We plan on having a similar "Knowledge Base" section like in WilsonAmplifiers.com. So we want to split each tab (knowledge base, how-to guide, case studies) into its own page "category", and put pages under each category.

Let me if this answers your question.

**Comment by live4js** [comment]
Unfortunately, the icons caused me some troubles and I couldn't finish all today. There are still some things which were left to do as: 

1. Include the new fonts. 
2. Make everything responsive.
3. Finish the design from Information & Resources.
4. And create the app which will extract the information from the store and will add it in the Information & Resources. 

First three points might take until Tuesday(possibly Wednesday), then we will need to see how to create the app which will generate the info in the Information & Resources panel.

**Comment by joelhenriquez23** [comment]
Ok i see... Laurentiu, we can hold off on the app for the Information & Resources panel for now. If the creating the app will take a high amount of hours we can hold off on doing this and we can add this app on Phase 2 of the site launch.

**Comment by joelhenriquez23** [comment]
Updated Homepage (In case Information & Resources is postponed):

https://drive.google.com/open?id=0BwElxc47J15_Skdib3U4c0MyTHc

**Comment by live4js** [comment]
Hi Joel,

I checked the PSD files and saw that you want to replace the Information & Resources panel with PowerReviews. 

Do you have the XML code for PowerReviews? I will need it for the home page to generate the content which you want. 

Thanks!

**Comment by joelhenriquez23** [comment]
Here is the FTP login where the XML file is stored:

partners.powerreviews.com | port 22
user: wilsona
pass: Fl7UaUib0N

The file name is feed.zip

**Comment by emilian** [comment]
SSH is running on that port, but I was logged out as soon as I entered the password.

Did the PowerReviews team add the IP address (109.74.197.202) to the whitelist?

**Comment by joelhenriquez23** [comment]
Ok...I sent a follow to Power Reviews to make sure that they whitelisted the IP address.

In the meantime, Simona asked for a sample of the XML file. Here is the url for that file: http://www.signalboosters.com/content/power-reviews-sample.xml

I'll give you an update once Power Reviews responds back.
